packages/@okta/vuepress-site/docs/guides/submit-app/overview/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Publishing an app integration with Okta Integration Network
+---
+
+This guide outlines the process by which software vendors (ISVs), Okta customers, and IT system-integrators (SI) can submit an app for review by Okta with the goal of publishing it to the Okta Integration Network (OIN).
```

Comment:
"independent software vendors"

Is this "SI" acronym actually used anywhere?

Reply:
Pulled